### ISSUE TITLE
Question -- styles vs convert markdown

### ISSUE THREAD

**Issue by mendelkranz** [iandol/scrivomatic#21]
Question: I noticed that Scrivener has an option to 'Convert Rich Text to Markdown' which basically attempts to do just that. Is there a benefit of using Styles, as you've indicated, over this option? Are there clear downsides to using this convert option? It seems both would export to the same markdown format. 

I'm thinking about this because I plan to also use scrivener extensively on iOS, and while it seems that styles might sync across, it's not possible to bind keyboard shortcuts to styles, and might make writing on iOS more difficult. 

Apologies if this is too far off the current repository.

**Comment by iandol** [comment]
The traditional problem was that `Convert Rich Text to Markdown` would also escape all other markdown, and break combining rich text and markdown syntax. BUT in a recent update you can disable escaping, which improves the ability to mix-and-match these two. I still prefer the semantic "purity" of using styles, but understand how clumsy it is to use on iOS at present, so you should be able to try it: **but** note you must then be careful not to use characters like [ ] and * * unless you want them to be markdown...

**Comment by mendelkranz** [comment]
I don’t use too much markdown aside from **bold** and *italics,* (in the humanities not the sciences). HOWEVER, I do use Pandoc, which uses [] for citations. Would that be a problem if trying to use `Convert Rich Text to Markdown`?

**Comment by iandol** [comment]
Yes, _by default_ `Convert Rich Text to Markdown` will escape Pandoc citations like so: `\[@shipp2013\]` — and therefore Pandoc cannot read them. **BUT** as I said before if you are using at least Scrivener V3.1.1 then this option was added, make sure it is OFF:
![Screenshot 2019-04-03 at 07 58 05](https://user-images.githubusercontent.com/153905/55443721-547fa000-55e6-11e9-95ad-8f876507a487.png)

Try to compile and see works for you...
